### PR TITLE
Replace "Anonymous" with Constant in `Const.java` #7932

### LINKED ISSUE #7932: Replace "Anonymous" with Constant in `Const.java`
Currently, we have a constant `Const.DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT` in `Const` class, but it can be renamed to make it more generic so that Constant can be used in all the classes not only in Comments Class.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes #7932

### CODE REVIEW

**Comment by sujeet14108** [comment]
Done

**Comment by sujeet14108** [comment]
@Mynk96 Ready for review :)

File: src/main/java/teammates/common/util/Const.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
     public static final String NO_SPECIFIC_RECIPIENT = "No specific recipient";
 
     public static final String DISPLAYED_NAME_FOR_SELF_IN_COMMENTS = "You";
-    public static final String DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT = "Anonymous";
+    public static final String ANONYMOUS_USER = "Anonymous";
```
Comment (Mynk96) [inline src/main/java/teammates/common/util/Const.java]: I think we should name the variable as `DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT`. `ANONYMOUS_USER` can be misunderstood as an object, other than that Looks good to be reviewed by a senior dev.

Comment (sujeet14108) [inline src/main/java/teammates/common/util/Const.java]: Done

**Comment by sujeet14108** [comment]
@chowyb Please review :)

**Comment by sujeet14108** [comment]
Done

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java [hunk 2]
```diff
@@ -869,7 +869,7 @@ private void testGetFeedbackSessionResultsForUser() throws Exception {
         actualStrings.add(nobodyUserName);
 
         // case: Anonymous User
-        String anonymousUserName = "Anonymous " + System.currentTimeMillis();
+        String anonymousUserName = Const.DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT + System.currentTimeMillis();
```
Comment (chowyb) [inline src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java]: Add the whitespace back on this line. Although the tests seem to be passing with this change, we should avoid modifying behaviour if possible.

Comment (sujeet14108) [inline src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java]: Done

**Comment by sujeet14108** [comment]
@chowyb Ready for review

**Pull request merged**